From: Roy Liu <carsomyr@gmail.com>
Date: Fri, 27 Mar 2015 15:14:35 -0400
Subject: Set the RBENV_ROOT environment variable to `/usr/lib/rbenv` instead
 of `${HOME}/.rbenv`

---
 libexec/rbenv | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libexec/rbenv b/libexec/rbenv
index 4880df3..4111437 100755
--- a/libexec/rbenv
+++ b/libexec/rbenv
@@ -51,7 +51,7 @@ else
 fi
 
 if [ -z "${RBENV_ROOT}" ]; then
-  RBENV_ROOT="${HOME}/.rbenv"
+  RBENV_ROOT="/usr/lib/rbenv"
 else
   RBENV_ROOT="${RBENV_ROOT%/}"
 fi
